lib/dor_indexing/indexers/descriptive_metadata_indexer.rb

```diff
@@ -21,15 +21,22 @@ def initialize(cocina:, **)
       def to_solr
         {
           # title
-          'sw_display_title_tesim' => title,
+          'main_title_tenim' => main_title, # for searching; 2 more field types are copyFields in solr schema.xml
+          'full_title_tenim' => full_title, # for searching; 1 more field type is copyField in solr schema.xml
+          'additional_titles_tenim' => additional_titles, # for searching; 1 more field type is copyField in solr schema.xml
+          'display_title_ss' => display_title, # for display in Argo
+          'sw_display_title_tesim' => display_title, # for display in Argo DEPRECATED in favor of display_title_ss
+
```

Comment:
these are the signficant changes -- adding 4 new title fields and marking the one existing title field as deprecated.